README.md

```diff
@@ -81,6 +81,36 @@ Entries that are nested within another Entry. Only Categories or Sub-Categories
 }
 ```
 
+### Deprecated node mapping
+When breaking changes such as deletion/collapsing of IDs or moving to a different category occur, the `deprecated_node_mapping` will serve as a reference to find the latest ids for deprecated nodes.
+
+**Any time such a breaking change occurs, it requires a new entry to be added to the deprecated_node_mapping**
```

Comment:
can we use hyphens for the name so it matches the file name:
`deprecated-node-mapping.json`